server/server_test.go

```diff
@@ -372,4 +412,49 @@ func testRunPartyInfo(t *testing.T, pi api.PartyInfo) {
 		t.Errorf("handler returned unexpected body: got %v wanted %v\n",
 			rr.Body.Bytes(), payload)
 	}
-}
\ No newline at end of file
+}
+
+func TestInit(t *testing.T) {
```

Comment:
This test does appear to be quite complex - are all of the stages really necessary?

Reply:
The most part of the test is to initialise various components, and now separated a small portion which was irrelevant here.